Makefile

```diff
@@ -75,24 +71,21 @@ community-page-check:
 	@echo "Started Community page tests."
 	@echo "Testing of community page data will be done here."
 	@echo "-----------"
-	cd CypressE2E && CYPRESS_BASE_URL=http://${FRONTEND_IP}:${FRONTEND_PORT}/ npm run Community_Tests
+	cd Cypress && CYPRESS_BASE_URL=$(URL) npm run Community_Tests
 
 .PHONY: e2e-metrics
 e2e-metrics:
 
 	@echo "----------------------------"
 	@echo "Pipeline Coverage Percentage"
 	@echo "----------------------------"
-	@sshpass -p ${portal_pass} ssh -o StrictHostKeyChecking=no ${portal_user}@${litmus_ip} -p ${port} -tt \
-	 "export CI_JOB_ID=${CI_JOB_ID} && export CI_PIPELINE_ID=${CI_PIPELINE_ID} && cd $(TESTPATH) && bash $(TESTPATH)/metrics/e2e-metrics"
+	bash ./metrics/e2e-metrics
 
 .PHONY: uninstall-portal
 uninstall-portal:
 
 	@echo "-----------"
 	@echo "Uninstalling Litmus-Portal"
 	@echo "-----------"
-	@sshpass -p ${portal_pass} ssh -o StrictHostKeyChecking=no ${portal_user}@${litmus_ip} -p ${port} -tt \
-	 "chmod 755 $(TESTPATH)/k8s_scripts/LitmusUninstall.sh"
-	@sshpass -p ${portal_pass} ssh -o StrictHostKeyChecking=no ${portal_user}@${litmus_ip} -p ${port} -tt \
-	 "$(TESTPATH)/k8s_scripts/LitmusUninstall.sh"
+	chmod 755 k8s_scripts/LitmusUninstall.sh
+	./k8s_scripts/LitmusUninstall.sh
```

Comment:
https://github.com/litmuschaos/litmus-e2e/pull/237/files#r583025340
for install as well.